cea/default.config

```diff
@@ -52,6 +52,42 @@ archetypes.type = MultiChoiceParameter
 archetypes.choices = comfort, architecture, HVAC, internal-loads, supply, restrictions
 archetypes.help = List of input databases to create.
 
+[streets-helper]
+bbox =
+bbox.type = ListParameter
+bbox.help = Default: empty, it uses district shapefile as bounding box, otherwise indicate Bounding box coordinates. bbox = min Longitude , min Latitude , max Longitude , max Latitude, if empty we
+
+streets = all_private
+streets.type = ChoiceParameter
+streets.choices = all_private, walk, bike, drive, drive_service, all
+streets.help = type of streets to gather. Default All_private
+
+[terrain-helper]
+grid-size = 30
+grid-size.type = IntegerParameter
+grid-size.nullable = false
+grid-size.help = Default: 30 m. grid size for raster. this affects the accuracy of reflections from the solar irradiation.
+
+elevation = 0
+elevation.type = RealParameter
+elevation.nullable = false
+elevation.help = Default: empty, leave empty to calculate it from satellite data. It will assume the same elevation for all the raster
+
+[district-helper]
+buffer = 50
+buffer.type = RealParameter
+buffer.help = Default: 50 meters, it is the length of the buffer area surrounding the Zone geometry
+
+height-ag =
+height-ag.type = RealParameter
+height-ag.nullable = true
+height-ag.help = Default: None, it is the average height of buildings. leave empty, we calculate it from Open Street Maps (no accurrate)
```

Comment:
These help texts show up in the dashboard interface. They need to be checked for grammar / spelling.

Reply:
fixed, thanks!